src/include/crowbar/complex.rb

```diff
@@ -730,7 +730,7 @@ def show_custom_repos_widget
     def InitRPRepos(id)
       # find the initial location now
       if @repos_location.empty?
-        @repos_location = "custom"
+        @repos_location = "local"
```

Comment:
I would like to improve this method but I'm not able to because I don't know what the format of the JSON files is. They should be linked from the README.